app/org/maproulette/framework/repository/UserRepository.scala

```diff
@@ -330,7 +395,11 @@ object UserRepository {
         ProjectManager(projectId, userId, osmId, displayName, avatarURL.getOrElse(""), roles)
     }
   }
-  private val standardColumns = "*, ST_AsText(users.home_location) AS home"
+  private val standardColumns = "*, ST_AsText(users.home_location) AS home, " +
+    "ARRAY(SELECT id from user_basemaps WHERE user_id=users.id) as custom_basemap_ids, " +
```

Comment:
So creating a select like this, even on an indexed key, could slow performance. So I would suggested modifying it to be like this: 
```sql
(SELECT ROW(
    ARRAY_AGG(id),
    ARRAY_AGG(name),
    ARRAY_AGG(url),
    ARRAY_AGG(overlay) 
FROM user_basemaps
WHERE user_id = users.id) AS custom_basemaps
```

This will return a record object for that row, so you would need to handle that differently in the parser, but would be a lot more efficient for the call to the database.